src/vdaf/prio3_test.rs

```diff
@@ -132,6 +132,7 @@ fn check_prep_test_vec<M, T, P, const SEED_SIZE: usize>(
     }
 }
 
+#[ignore] // TODO(issue #477)
```

Comment:
@tgeoghegan Let me know if you want me to remove these TODOs.

Reply:
I'm happy enough with the issue references